packs/libcloud/actions/create_vm.py

```diff
@@ -32,11 +31,10 @@ def run(self, credentials, name, size_id=None, image_id=None, size_name=None, im
             size = [s for s in driver.list_sizes() if size_name in s.name][0]
 
         if image_id is not None:
-            image = NodeImage(id=image_id, name=None,
-                              driver=driver)
+            image = [i for i in driver.list_images() if image_id == i.id][0]
```

Comment:
Ah, good point.

And yeah, using `list_images` tends to be safer, but in some cases it adds a lot of overhead (list_image responses can be massive) so if you already know the id and want to avoid additional request that's the preferred approach.